test/test_enzyme.jl

```diff
@@ -319,9 +319,9 @@ end
     Δν = 1e-6
     ν1 = ν₀ + Δν
     ν2 = ν1 + Δν
-    e1 = viscous_hydrostatic_turbulence(ν1, model, u_init, v_init, Δt, u_truth, v_truth)
+    e1 = viscous_hydrostatic_turbulence(ν₀, model, u_init, v_init, Δt, u_truth, v_truth)
     e2 = viscous_hydrostatic_turbulence(ν2, model, u_init, v_init, Δt, u_truth, v_truth)
-    ΔeΔν = (e2 - e1) / Δν
+    ΔeΔν = (e2 - e1) / 2Δν
```

Comment:
ν2 - ν1 = Δν right?

Reply:
I'm using central difference here, maybe I should rename e1 to e0. But it's (v1 + delta v) - (v1 - delta v)

Reply:
Ok, but in that case there is a typo

Reply:
I made a few suggested changes